doc/users/dflt_style_changes.rst

```diff
@@ -473,6 +473,36 @@ or by setting::
 in your :file:`matplotlibrc` file.
 
 
+``fill_between`` and ``fill_betweenx``
+--------------------------------------
+
+`~matplotlib.axes.Axes.fill_between` and
+`~matplotlib.axes.Axes.fill_betweenx` both follow the patch color
+cycle.
```

Comment:
Good, that makes sense.  This just shows that I have not followed the development of the cycling code enough to have a clear picture of it in my head, and I didn't look it up.